vimeo-networking/src/main/java/com/vimeo/networking/VimeoClient.java

```diff
@@ -137,13 +143,28 @@ private VimeoClient(@NotNull Configuration configuration) {
         setVimeoAccount(vimeoAccount);
     }
 
+    /**
+     * Sets a new base URL to be used for requests
+     * by the VimeoClient for specific paths. If a null
+     * base URL is provided, the default base URL will be
+     * used by the client.
+     *
+     * @param path    the path for which to change the base URL.
+     * @param baseUrl the new base URL to use, may be null.
+     */
+    public void setBaseUrlForRequest(@NotNull String path, @Nullable String baseUrl) {
+        mBaseUrlInterceptor.setBaseUrlForRequest(path, baseUrl);
+    }
```

Comment:
Also being able to clear this url map to go back to the default would be beneficial - I'm looking at this from an "Admin Panel" perspective with a "Reset to defaults" type option.